frontend/controller/actions/group.js

```diff
@@ -561,6 +563,32 @@ export default (sbp('sbp/selectors/register', {
       sbp('okTurtles.events/emit', REPLACE_MODAL, 'IncomeDetails')
     }
   },
+  'gi.actions/group/checkGroupSizeAndProposeMember': async function () {
+    // if current size of the group is >= 150, display a warning prompt first before presenting the user with
+    // 'AddMembers' proposal modal.
+
+    const { groupMembersCount } = sbp('state/vuex/getters')
+    const isGroupSizeLarge = groupMembersCount >= MAX_GROUP_MEMBER_COUNT
+
+    if (isGroupSizeLarge) {
+      const promptOpts = {
+        heading: 'Large group size',
+        question: L(
+          "Groups over 150 members are at significant risk for fraud, {a_}because it is difficult to verify everyone's identity.{_a} Are you sure that you want to add more members?",
+          { a_: `<a class='link' href='${ALLOWED_URLS.WIKIPEDIA_DUMBARS_NUMBER}'' target='_blank'>`, _a: '</a>' }
+        ),
+        yesButton: L('Yes'),
+        noButton: L('Cancel')
```

Comment:
I don't remember for sure, but I think `yesButton` is the default right? If so, per the edited comment in the issue, we want the default to say `Cancel`, not `Yes`